### DISCUSSION TITLE
Detached and Leader

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#7716] [Q&A] [answered]
Hello,

From what I understand, when a node state is "detached", it means that it is not in a Thread network. I've noticed that sometimes my full-thread device becomes detached, and then becomes child (i.e., re-joins the network). What causes this to happen?

I also noticed that some of the devices have their node state as leader, which from my understanding meant that they created their own partition. What causes units to create their own partition? If it is due too poor connectivity, I assume there is a signal quality threshold that OpenThread uses for this? Is it possible for me to modify it and maybe lower the threshold?

**Comment by jwhui** [comment, marked as answer]
Whenever Thread devices lose fail to maintain connectivity with their existing Thread partition, they will detach and start the attach procedure to look for neighboring devices. Thread requires all devices to initially attach as end devices, then later upgrade to a router. A child detaches when it is not able to communicate with its parent. A router detaches from the partition when it is not able to receive periodic updates from the Leader via MLE Advertisements.

A router-capable device will become a Leader and form its own partition if it cannot discover any neighboring router-capable devices during the attach process. During the attach process, there are no signal quality thresholds. If the device is not able to attach as a child, it means it was not able to complete the attach process with any neighboring nodes.
